### ISSUE TITLE
works not with the quobuz web player although html5 playback is selected

### ISSUE THREAD

**Issue by ChristophHeilmann** [ejci/Chrome-Audio-EQ#15]

**Comment by ejci** [comment]
Does it work now with 0.3.2.x version?

**Comment by ChristophHeilmann** [comment]
No, I own a Chromebook with Chrome OS and  this system use the Chrome Audio Server (CRAS). The Chrome Audio Server has an inbuild DSP which could activated by a website. See http://events.linuxfoundation.org/sites/events/files/slides/CRAS%20ELC%202014%20(1).pdf
and http://audio-tuning.appspot.com/.
 
But this works only with speakers and until now I haven't found a possibility to use the EQ with an external DAC.

**Comment by ejci** [comment]
Hi,
http://audio-tuning.appspot.com/ is doing exactly the same thing as this extension. Both are using HTML5 audio API to manipulate the sound. The only difference is that in the extension Im attaching EQ to all HTML5 video/audio sources on all pages. 

This will never work with external DAC.

**Comment by ChristophHeilmann** [comment]
I suppose that  the CRAS EQ is different. You need a python script to convert the EQ config file into a CRAS dsp.ini file, see https://chromium.googlesource.com/chromiumos/third_party/adhd/+/2ab4ecc17852ae6e95b398d2fac45791e105488f

**Comment by ejci** [comment]
But thats something different. You export the "config file" and then use it on OS level. It has nothing to do with chrome.

**Comment by ChristophHeilmann** [comment]
Yes, that is what I've said. I don't know if its possible to make a chrome OS app from it because if you change the EQ settings you always have to rebuild CRAS. Here you can see the archtecture of CRAS: http://freepine.blogspot.co.at/2014/06/introduction-of-audio-pipeline-in.html
